pod/perldelta.pod

```diff
@@ -91,16 +91,12 @@ as an updated module in the L</Modules and Pragmata> section.
 
 =head1 Performance Enhancements
 
-XXX Changes which enhance performance without changing behaviour go here.
-There may well be none in a stable release.
-
-[ List each enhancement as an =item entry ]
-
 =over 4
 
 =item *
 
-XXX
+Additional optree optimizations for common OP patterns.
+L<[GH #19943]|https://github.com/Perl/perl5/pull/19943>.
```

Comment:
Might be worth expanding on that a little more, just to say something like "common patterns of multiple small ops are combined into fewer larger ones, so as to be more efficient at runtime".

Reply:
I've elaborated.